apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -265,6 +267,7 @@ export class LemonadePipeline implements BasePipeline {
                     )} , pipeline '${this.id}'`;
                     logs.push(makePLogWarn(message));
                     logger(`${LOG_TAG} ${message}`);
+                    atomsExpected--;
                   }
                 } else {
                   const message = `Unsupported ticket type ${
```

Comment:
Do we need to do atomsExpected-- in this case as well?

Reply:
Same for the block for parsing error. I wonder if it's easier just to not add it before the for loop and do ++ on the success case.

Reply:
I'm kind of cheating here with the meaning of `atomsExpected`, let's keep it this way and see how we like it